kratos/processes/calculate_discontinuous_distance_to_skin_process.cpp

```diff
@@ -38,7 +41,8 @@ namespace Kratos
         , mrSkinPart(rSkinPart)
         , mrVolumePart(rVolumePart)
         , mOptions(CalculateDiscontinuousDistanceToSkinProcessFlags::CALCULATE_ELEMENTAL_EDGE_DISTANCES.AsFalse()
-             | CalculateDiscontinuousDistanceToSkinProcessFlags::CALCULATE_ELEMENTAL_EDGE_DISTANCES_EXTRAPOLATED.AsFalse())
+             | CalculateDiscontinuousDistanceToSkinProcessFlags::CALCULATE_ELEMENTAL_EDGE_DISTANCES_EXTRAPOLATED.AsFalse()
+             | CalculateDiscontinuousDistanceToSkinProcessFlags::USE_NEGATIVE_EPSILON_FOR_ZERO_VALUES.AsFalse())
```

Comment:
Just wondering, why setting it to false by default?

Reply:
I was trying to be consistent with the rest of flags. This way if someone wants the NEGATIVE_EPISLON, the flag defition is done as true, which seems cleaner:

https://github.com/KratosMultiphysics/Kratos/blob/a1a5723c6b61c542bc4d8788f87758370ee92efe/kratos/tests/cpp_tests/processes/test_calculate_discontinuous_distance_to_skin_process.cpp#L334-L337

Reply:
I think I've misunderstood the flag purpose. Which is the correct one?
Option 1: The flag switches on/off the zero values check
Option 2: The check is always on but the flag says if the epsilon is positive or negative.

Reply:
> I think I've misunderstood the flag purpose. Which is the correct one?
> Option 1: The flag switches on/off the zero values check
> Option 2: The check is always on but the flag says if the epsilon is positive or negative.

Option 2. There is always an epsilon (as we discussed in person). 


Note that some commits ago I defined two flags, one to control the zero check (option 1) and the other one to control the sign (option 2). Since option 1 requires to do some postprocessing from the user to decide which side is intersected or not, we decided to always do the zero check and the epsilon replacement.

Reply:
You're right. Now I remember discussing this in person.

My point was that the current behavior is a positive epsilon, but to achieve this we do `USE_NEGATIVE_EPSILON.AsFalse()`. For me it would be much more intuitive to do `USE_POSITIVE_EPSILON`, although this is not consistent with the other flags.

Reply:
I see your point. You are right it feels more intuitive to have the USE_POSITIVE_EPSILON flag, I'll change it